docs/src/pages/guides/examples.md

```diff
@@ -0,0 +1,249 @@
+        "username": "YOUR_GMAIL_ACCOUNT",
+        "password": "YOUR_GMAIL_PASSWORD_OR_APP_PASSWORD"
+      }
+    }
+  ],
+  "probes": [
+    {
+      "id": "1",
+      "name": "Monika Landing Page",
+      "description": "Landing page of awesome Monika",
+      "interval": 10,
+      "requests": [
+        {
+          "url": "https://hyperjumptech.github.io/monika",
+          "timeout": 7000
+        }
+      ],
+      "alerts": ["status-not-2xx"]
+    }
+  ]
+}
+```
+
+Using the above configuration, Monika will check the landing page every 10 seconds and will send a notification by email when the landing page is down 5 times in a row. For more information about available notification channels, refer to [Notifications](https://hyperjumptech.github.io/monika/guides/notifications).
+
+## HTML Form Submission Example
+
+Here is probe example with POST request to simulate HTML form submission
+
+```json
+{
+  "probes": [
+    {
+      "id": "1",
+      "name": "HTML form submission",
+      "description": "simulate html form submission",
+      "interval": 10,
+      "requests": [
+        {
+          "method": "POST",
+          "url": "http://www.foo.com/login.php",
+          "timeout": 7000,
+          "headers": {
+            "Content-Type": "application/x-www-form-urlencoded"
+          },
+          "body": {
+            "username": "someusername",
+            "password": "somepassword"
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+Using the configuration above, Monika will send a POST request to http://www.foo.com/login.php with the defined request's body.
+
+## Multiple request
+
+Here is an example configuration with multiple requests:
+
+```json
+{
+  "probes": [
+    {
+      "id": "1",
+      "name": "Probing Github",
+      "description": "simulate html form submission",
+      "interval": 10,
+      "requests": [
+        {
+          "method": "GET",
+          "url": "https://github.com/",
+          "timeout": 7000
+        },
+        {
+          "method": "GET",
+          "url": "https://github.com/hyperjumptech",
+          "timeout": 7000
+        }
+      ],
+      "incidentThreshold": 3,
+      "recoveryThreshold": 3,
+      "alerts": ["status-not-2xx", "response-time-greater-than-200-ms"]
+    }
+  ]
+}
+```
+
+In the configuration above, Monika will first check `https://github.com/` then `https://github.com/hyperjumptech`. If the status code of `https://github.com/` is not 2xx (e.g., 200, 201), Monika **will not** check `https://github.com/hyperjumptech`.
+
+If there is a case where executing GET request to `https://github.com` triggered an alert, the next request will not be executed.
+
+## Requests Chaining
+
+Monika supports requests chaining, which enables you to do multiple request and use previous request(s) response for other request. For example, after executing a GET request to certain API, the next request could use the previous request(s) response into their path/query parameters or headers.
```

Comment:
Monika supports request chaining, which enables you to do multiple requests and the ability to use past responses from earlier requests. (s for tenses + compunds IS a pain in the ass)